effekt/jvm/src/test/scala/effekt/RecursiveDescentTests.scala

```diff
@@ -125,6 +125,9 @@ class RecursiveDescentTests extends munit.FunSuite {
     parseExpr("fun() { foo(()) }")
 
     parseExpr("10.seconds")
+
+    parseExpr("[1,2,3]")
+    parseExpr("[3,2,1,]")
```

Comment:
Is it also possible to add some tests for an empty and a singleton list with trailing commas?
Just to make sure that even the trivial cases work :)